plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -259,14 +259,14 @@ class StaticHtmlReporter : Reporter {
 
                 tbody {
                     ruleViolations.forEachIndexed { rowIndex, ruleViolation ->
-                        evaluatorRow(rowIndex + 1, ruleViolation)
+                        ruleViolationRow(rowIndex + 1, ruleViolation)
```

Comment:
For the row I believe singular is fine, as it may only contain a single violation.